### ISSUE TITLE
Using { force: true } should disable attempting to scroll the element into view

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#553]
When applying an action such as `click` or `type`, our interaction algorithm will attempt to the scroll the element into view and continuously "nudge it".

This obviously still needs some work as users are experiencing situations where the element is still covered up in a container that is not scrollable such as #537.

The simplest fix here is just to disable scrolling an element into view when used with `{ force: true }` and remove this error checking.

**Comment by brian-mann** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by mediavrog** [comment]
Was this released? I was trying

```
cy.get('[data-role="input-title"]')
      .clear()
      .type(this.interviewFixture.title, { delay: 50, force: true });
```

and the input scrolled to the top, even though it was visible before.

**Comment by jennifer-shehane** [comment]
Um, me neither 😳

**Comment by hkarambizi** [comment]
@brian-mann @jennifer-shehane: I have been browsing through all issues related to this. I also read the release notes for 0.20.0. Is adding `{force: true}` the only resolution to the .click() and .type() scrolling and getting covered by header/navigation DOM elements? I worry about how this will add to test flakiness. 

A large amount of what I need to test involves actions that would get covered up. I have an example using .type() below.
I added a cy.pause() to debug and look exactly at what point the browser scrolls.
Here is a animated GIF: https://cl.ly/a43de8db9f6e

Please advise if there were any updates since. Otherwise I will continue to add force: true.

**Comment by jennifer-shehane** [comment]
No, there have not been any updates beyond this. Using `{force: true}` is the recommended workaround. If you're worried that the element is not 'ready' in some way, I would add assertions about the input being visible / not disabled / or whatever before doing the type with force.

**Comment by skydever** [comment]
Hi! I am still having issues using `{force: true}` as a workaround. the click will be executed but there must be some kind of check after the click, at least it seems like? The error message also tells me to use `{force: true}`, but I already do ...

I see the DOM changes when I hover the click entry, but it is still failing:

![issue-553](https://user-images.githubusercontent.com/15961578/64327818-a16c0600-cfcc-11e9-883e-3a80b0be6060.png)

**Comment by jennifer-shehane** [comment]
This was released in [Cypress 0.20.0](https://docs.cypress.io/guides/references/changelog.html#0-20-0) 
This issue will be closed to further comment as the exact issue here was resolved and tested. 

@skydever If you're experiencing a bug similar to this in Cypress, please [open a new issue](https://github.com/cypress-io/cypress/issues/new) with a fully reproducible example that we can run. There may be a specific edge case with the issue that we need more detail to fix.
